components/k8s-model-server/iap-client/get_token.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
```

Comment:
Per comment above. Can we create a generic python script that uses the GCP Python client library to get a credential.  Then accept a couple command line parameters like
--endpoint=https://some-endpoint/my-model/model-x/
--payload=/tmp/payload.json

Reply:
Done